### ISSUE TITLE
Bad autogenerated translations for supported languages (i18n), placeholders are not processed

### ISSUE THREAD

**Issue by Juraj-Masiar** [mozilla/addons#1441]
### Describe the problem and steps to reproduce it:
When you upload new addon that supports multiple languages, the page will automatically generate local-specific descriptions for them:

![Screenshot 2022-10-17 at 14-01-15 Edit Product Page Search Result Previews Add-ons for Firefox](https://user-images.githubusercontent.com/6358971/196173326-d9ca4831-6b37-4cdf-8604-6c9eaf2046d9.png)

### What happened?
However, these has bad auto-generated fields, specifically if you are using "placeholders", those will not be correctly replaced:
```json
  "app_desc": {
    "message": "$app_name$ shows you page previews next to each search result.",
    "placeholders": {
      "app_name": {
        "content": "Search Result Previews"
      }
    }
  }
```
This will auto-generate following field in the "Slovak" language:

![Screenshot 2022-10-17 at 14-00-47 Search Result Previews – rozšírenie pre 🦊 Firefox (sk)](https://user-images.githubusercontent.com/6358971/196173796-4453f34f-5c21-4ff8-acb2-8cdd1c2b04ae.png)

### What did you expect to happen?
Placeholders should be correctly processed.

**Comment by diox** [comment]
Yes, placeholders are not processed at all at the moment - they are pretty rare in the fields we extract (`name`, `homepage`, `summary`). Patches welcome to implement that though.

Relevant documentation: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/i18n/Locale-Specific_Message_reference#placeholders

Relevant code in AMO (ran when creating/validating the `Addon`): https://github.com/mozilla/addons-server/blob/e7d7fd0e11a220098bf9723c997dd5e230c59546/src/olympia/files/utils.py#L1126-L1167
